Makefile

```diff
@@ -1,10 +1,11 @@
 .PHONY: build run lint test
+PORT ?= 5002
 
 build:
 	poetry run ./scripts/build.sh
 
 stop-ajv:
-	npm run stop
+	@PORT=$(PORT) npm run stop
 
 start-ajv:
 	npm run start
```

Comment:
`start-ajv` also needs the PORT var?

Reply:
START AJV runs app.js which fetches the PORT from the `.env` file. Unlike STOP AJV which has to hard code the port  it wants to stop